samples/SortBenchmark/qsort/qsort.cpp

```diff
@@ -35,10 +34,10 @@ bool compareb(const void * a, const void * b)
 }
 
 extern "C" __declspec(dllexport)
-void stdqsort(void* buf, int len, int dim)
+void stdquicksort(void* buf, int len, int dim)
 {
 	dim = d;
-	qsort(buf, len, dim, compare);
+	qsort_s(buf, len, dim, compare, null);
```

Comment:
"extern "C"" makes a function name in C++ have 'C' linkage (thus no name mangling).  But this code is still C++, right?